src/modules/launcher/Wox.Infrastructure/Hotkey/HotkeyModel.cs

```diff
@@ -82,7 +82,7 @@ private void Parse(string hotkeyString)
                 return;
             }
 
-            List<string> keys = hotkeyString.Replace(" ", string.Empty).Split('+').ToList();
+            List<string> keys = hotkeyString.Replace(" ", string.Empty, StringComparison.Ordinal).Split('+').ToList();
```

Comment:
I think these are related to key codes so maybe it's better to use invarient here?